dp/cloud/python/magma/configuration_controller/tests/unit/test_response_processor.py

```diff
@@ -434,19 +444,24 @@ def _create_db_requests(
     def _get_db_enum(self, data_type, name):
         return self.session.query(data_type).filter(data_type.name == name).first()
 
-    def _prepare_response_from_db_requests(self, db_requests, response_code=None):
+    def _prepare_response_from_db_requests(self, db_requests, response_code=0, response_data=None):
         req_type = db_requests[0].type.name
         response_payload = self._create_response_payload_from_db_requests(
             response_type_name=request_response[req_type],
             db_requests=db_requests,
+            sas_response_code=response_code,
+            sas_response_data=response_data,
         )
-        return self._prepare_response_from_payload(req_type, response_payload, response_code)
+        return self._prepare_response_from_payload(req_type, response_payload, response_code, response_data)
 
     @staticmethod
-    def _prepare_response_from_payload(req_type, response_payload, response_code=None):
-        if response_code is not None:
+    def _prepare_response_from_payload(req_type, response_payload, response_code=0, response_data=None):
```

Comment:
Why we changed response code's default value from `None` to `0`? It looks like it changes semantic of this functions, since we check whether response code is `None`, so for callers which did not specify this parameter, the behavior is changed